tinygrad/tensor.py

```diff
@@ -245,6 +249,9 @@ def dispatch(*x, **kwargs):
     return f.apply(f, *x, **kwargs)
   setattr(Tensor, name, dispatch)
   # TODO: div is a second class op, so it doesn't work here
+  if not hasattr(Tensor, "__neg__"):
```

Comment:
Indentation. And this runs for every name.

Reply:
uhm.. but will add __neg__ once..